src/java.base/linux/classes/sun/nio/fs/LinuxFileStore.java

```diff
@@ -142,19 +142,9 @@ public boolean supportsFileAttributeView(Class<? extends FileAttributeView> type
             }
 
             // user_xattr option not present but we special-case ext4 as we
-            // know that extended attributes are enabled by default for
-            // kernel version >= 2.6.39
+            // know that extended attributes are enabled by default
             if (entry().fstype().equals("ext4")) {
-                if (!xattrChecked) {
-                    // check kernel version
-                    int[] kernelVersion = getKernelVersion();
```

Comment:
Can getKernelVersion be removed too?

Reply:
There's probably some imports that that be removed once the Pattern code is removed.